starknet_py/tests/e2e/docs/guide/test_resolving_proxies.py

```diff
@@ -10,6 +10,9 @@
 from starknet_py.proxy.proxy_check import ArgentProxyCheck, ProxyCheck
 
 
+@pytest.mark.skip(
+    reason="This functionality is deprecated. Should be removed in the future."
```

Comment:
```suggestion
    reason="Proxies are not supported in Cairo 1."
```
Or something similar